src/untils/users.js

```diff
@@ -0,0 +1,7 @@
+const usersServices = require('../services/users/users.services');
+
+const isUserExist = (userId) => {
+  return !!(userId && !isNaN(userId) && usersServices.getUserById(userId));
```

Comment:
The check `userId && !isNaN(userId)` is slightly redundant. Since `isNaN(userId)` will return true for non-numeric values, the `userId` check is not necessary. Consider simplifying this condition to `!isNaN(userId) && usersServices.getUserById(userId)`.